src/components/graph/GraphCanvas.vue

```diff
@@ -298,6 +287,68 @@ watch(
   { immediate: true }
 )
 
+// Ensure slot layout sync starts whenever a canvas is available (LiteGraph mode)
+watch(
+  () => canvasStore.canvas,
+  (canvas, oldCanvas) => {
+    if (!canvas) {
+      // Canvas was removed - stop sync if active
+      if (slotSync && slotSyncStarted) {
+        slotSync.stop()
+        slotSyncStarted = false
+      }
+      // Clear any stale slot layouts when canvas is torn down
+      layoutStore.clearAllSlotLayouts()
+      return
+    }
+
+    // Canvas changed - restart sync
+    if (oldCanvas && oldCanvas !== canvas) {
+      if (slotSync && slotSyncStarted) {
+        slotSync.stop()
+        slotSyncStarted = false
+      }
+    }
```

Comment:
Might be worth extracting some names here, but otherwise I'd just rather avoid unnecessary nesting.
```suggestion
    if (oldCanvas && oldCanvas !== canvas && slotSync && slotSyncStarted) {
        slotSync.stop()
        slotSyncStarted = false
    }
```